tests/test_datamodel_service.py

```diff
@@ -143,6 +143,41 @@ def test_add_on_affected(new_mesh_session):
     sleep(5)
     assert len(data) > 0
     assert data[0] == True
+
+    calls = []
+    subscription2 = meshing.workflow.add_on_affected(lambda obj: calls.append(True))
+    geom = examples.download_file(
+        filename="mixing_elbow.pmdb", directory="pyfluent/mixing_elbow"
+    )
+    import_geom = meshing.workflow.TaskObject["Import Geometry"]
+    assert "FileName" not in import_geom.Arguments()
+    assert import_geom.CommandArguments()["FileName"] is None
+    import_geom.Arguments = {"FileName": geom}
+    assert import_geom.Arguments()["FileName"] == geom
+    assert import_geom.CommandArguments()["FileName"] == geom
+    sleep(1)
+    assert calls == [True]
+    import_geom.Arguments = {"FileName": "dummy"}
+    sleep(1)
+    assert calls == 2 * [True]
+    import_geom.Arguments = {"FileName": geom}
+    sleep(1)
+    assert calls == 3 * [True]
+    execute_state = [meshing.workflow()]
+    import_geom.Execute()
+    calls_after_execute = []
+    loop_count = 10
+    for i in range(loop_count):
+        sleep(5)
```

Comment:
Are there any bools or objects that we could check here to minimize the amount of time wasted sleeping? Perhaps check if a new `bool` has been added to the `calls` list, or until list length has increased? Could perhaps use one of these for convenience after they are merged: https://github.com/ansys/pyfluent/pull/1748#discussion_r1252256494